content/guides/getstarted/setup-local-development.md

```diff
@@ -38,48 +38,266 @@ deploying Solana programs to the blockchain.
+Are YOU ready though? Let’s start.
 
-- [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install),
-  which provides a Linux environment that runs inside your existing Windows
-  installation. You can then run regular Linux software, including the Linux
-  versions of Solana CLI.
-- [Windows Terminal](https://www.microsoft.com/store/productId/9N0DX20HK701), a
-  modern terminal for Windows.
+Choose which operating system you’ll be using to build on Solana.
+Mainly We will cover three operating systems, Windows (with/without WSL), Linux and MacOS.
 
-Once you've installed both Windows Subsystem for Linux and Windows Terminal,
-open Windows Terminal and start a new tab for your Linux distribution (e.g.
-'Ubuntu') using the `wsl` command.
+This guide has seven sections
 
-Then continue with this guide, running each of the terminal commands within this
-Linux terminal.
+1. Install Dependencies
+2. Install rust and cargo toolchain
+3. Install Solana cli 
+4. Install Anchor
+5. Setup a local blockchain cluster
+6. Create a filesystem wallet
+7. Airdrop Solana tokens to your wallet
 
-## Install the Solana CLI
+## 1. Installing Dependencies
 
-To interact with the Solana clusters from your terminal, install the
-[Solana CLI tool suite](https://docs.solana.com/cli/install-solana-cli-tools) on
-your local system:
+Just because rust compiles and builds your software into binary that can run for the computer architecture we specify, we need to install some OS level dependencies on our machine. 
 
-```bash
-sh -c "$(curl -sSfL https://release.solana.com/stable/install)"
+### Let’s start with Windows 8/10/11
+
+There are two ways to get started on windows, 
+
+1.  build on windows
+2.  build on windows but with WSL
+
+WSL is windows subsystem for linux, which translates Linux system calls into Windows system calls using a compatibility layer, **for running Linux binary executables natively** and it runs Linux processes alongside the Windows kernel in a lightweight virtual machine. 
+
+WSL and linux will mostly have same steps to go through.
+
+### Build Dependencies for Windows
+
+1. Rust requires build tools to compile for our Solana Tools.
+Download the build tools from here - 
+choose C/C++ build tools by minimum
+
+When you download, make sure to run the tool as an administrator (right click on executable and choose run as administrator)
+    
+    [Download Visual Studio Tools - Install Free for Windows, Mac, Linux](https://visualstudio.microsoft.com/downloads/?q=build+tools#build-tools-for-visual-studio-2022)
+    
+2. Now Open a Command Prompt (`cmd.exe`) as an Administrator
+    
+    ![setup—-windows.exe](/assets/guides/setup-local-environment/setup---rustup-windows.png)
+    
+---
+
+### Build Dependencies for WSL
+
+1. Install WSL in your system, make sure to restart your computer when installing is done.
+    
+    [Install WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
```

Comment:
Good stuff - better to link to current instructions like you're doing than to maintain our own, out of date, set of instructions.